content/maintainers-guide/project-organization.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Project organization
+weight: 1
+---
+
+# Project organization
+
+## Project roles
+
+### Product Manager
+
+Product Managers provide the project's product vision, ensuring that the software provides an answer to the needs of its different types of users.
+
+Current product managers:
+
+- Fabien Vallon ([@fabienvallon](https://github.com/fabienvallon))
+- Mateus Shirlaw ([@MatShir](https://github.com/MatShir))
+
+### Maintainer
+
+Maintainers provide the project's technical vision, review code contributions, and ensure that the software develops in line with the product vision.
+
+Current maintainers:
+
+- Franck Lefèvre ([@Progi1984](https://github.com/Progi1984))
+- Ibrahima Sow ([@sowbiba](https://github.com/sowbiba))
+- Jonathan Lelièvre ([@jolelievre](https://github.com/jolelievre))
+- Krystian Podemski ([@kpodemski](https://github.com/kpodemski))
+- Mathieu Ferment ([@matks](https://github.com/matks))
+- Matthieu Rolland ([@matthieu-rolland](https://github.com/matthieu-rolland))
+- Pablo Borowicz ([@eternoendless](https://github.com/eternoendless))
+- Pierre Rambaud ([@PierreRambaud](https://github.com/PierreRambaud))
+- Pululu Kinanga André ([@PululuK](https://github.com/PululuK))
+- Thomas Baccelli ([@atomiix](https://github.com/atomiix))
+- Valentin Szczupak ([@NeOMakinG](https://github.com/NeOMakinG))
+
+### UX Designer
+
+UX Designers make sure that interfaces follow the best practices in usability and design.
+
+Current UX designers:
+
+- Scott Cosio ([@prestascott](https://github.com/prestascott))
+- Tristan Lehot ([@TristanLDD](https://github.com/TristanLDD))
+
+### Quality Assurance (QA)
+
+Issue managers qualify new issues to avoid duplicates and, if they are indeed defects in the software, their severity.
+
+Current issue managers:
```

Comment:
```suggestion
### Issue Managers

Issue managers reproduce and qualify new issues to avoid duplicates and, if they are indeed defects in the software, their severity. They also redirect new improvement suggestions to the right person in the PM team.

Current issue managers:
```